src/libraries/System.Text.Json/gen/src/System.Text.Json.SourceGeneration.csproj

```
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <CLSCompliant>false</CLSCompliant>
+    <Nullable>enable</Nullable>
+    <!-- Suppress warning XML comment has cref attribute that could not be resolved -->
+    <NoWarn>CS1574</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);BUILDING_SOURCE_GENERATOR</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(MicrosoftCodeAnalysisCSharpWorkspacesVersion)" PrivateAssets="all" />
```

```suggestion
    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(MicrosoftCodeAnalysisCSharpWorkspacesVersion)" PrivateAssets="all" />
    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Packaging" Version="$(MicrosoftDotNetBuildTasksPackagingVersion)" PrivateAssets="all" />
```
This is needed as you had it here https://github.com/dotnet/runtimelab/blob/33815215fa7c2ac5c664159d651314c3d86aed17/src/libraries/System.Text.Json/System.Text.Json.SourceGeneration/System.Text.Json.SourceGeneration.csproj#L16

Can be removed when we get rid of pkgproj.

---

Thanks!